src/Illuminate/Support/helpers.php

```diff
@@ -559,6 +559,27 @@ function head($array)
     }
 }
 
+if (! function_exists('image_data_uri')) {
+    /**
+     * Convert an image to data URI.
+     * 
+     * @param  string $image
+     * @return string
```

Comment:
You can do your `return` directly in the `if()` statement and get rid of the `$data_uri` variable. No return statement is also `NULL`.